### PR TITLE
updated flag descriptions

### PR DESCRIPTION
**Pull request by hellt**

### CODE REVIEW

File: cmd/prompt.go [hunk 1]
```diff
@@ -136,9 +136,9 @@ var promptDirs []string
 
 // used to init or reset pathCmd flags for gnmic-prompt mode
 func initPromptFlags(cmd *cobra.Command) {
-	cmd.Flags().StringArrayVarP(&promptFiles, "file", "", []string{}, "yang files to get the paths")
-	cmd.Flags().StringArrayVarP(&promptExcluded, "exclude", "", []string{}, "yang modules to be excluded from path generation")
-	cmd.Flags().StringArrayVarP(&promptDirs, "dir", "", []string{}, "directories to search yang includes and imports")
+	cmd.Flags().StringArrayVarP(&promptFiles, "file", "", []string{}, "path to yang files to get auto-completions from")
```
Comment (karimra) [inline cmd/prompt.go:139]: maybe add that it can be a dir as well

File: cmd/prompt.go [hunk 2]
```diff
@@ -136,9 +136,9 @@ var promptDirs []string
...
+	cmd.Flags().StringArrayVarP(&promptExcluded, "exclude", "", []string{}, "yang module names to be excluded from auto-completion generation")
```
Comment (karimra) [inline cmd/prompt.go:140]: I would add "... excluded from __path__ auto-completion generation"

File: cmd/prompt.go [hunk 3]
```diff
@@ -136,9 +136,9 @@ var promptDirs []string
...
+	cmd.Flags().StringArrayVarP(&promptDirs, "dir", "", []string{}, "path to directory with yang files used as includes and/or imports")
```
Comment (karimra) [inline cmd/prompt.go:141]: shouldn't it be "yang __modules__ used as includes..." ?

File: cmd/prompt.go (hunk 2 above)
Comment (hellt) [reply cmd/prompt.go:140]: Initially I on purpose removed the path, as we do (plan to?) extract more than just path information, like types.

But if that causes confusion, lets revert it to having **path** in the auto-completion. To be consistent, the **path** should be added to --file as well

File: cmd/prompt.go (hunk 1 above)
Comment (hellt) [reply cmd/prompt.go:139]: fair, will do

**Comment by hellt** [comment]
@karimra addressed your comments

**Review by karimra — APPROVED** [review]

**Comment by karimra** [comment]
lgtm

**Pull request merged**